tests/system/tsystem_misc.nim

```diff
@@ -63,6 +63,16 @@ doAssert high(float64) > low(float64)
 var s = @[1]
 s.delete(0)
 
+# deleting nonexistent index should raise #16544
```

Comment:
```suggestion
# bug #16544: deleting out of bounds index should raise
```
(see contributing.html for rationale for having a fixed formatting for issue references for ease of search)